dynamiq/connections/connections.py

```diff
@@ -932,6 +933,44 @@ def connect(self):
         pass
 
 
+class PGVector(BaseConnection):
+    type: Literal[ConnectionType.PGVector] = ConnectionType.PGVector
+    host: str = Field(default_factory=partial(get_env_var, "PGVECTOR_HOST", "localhost"))
```

Comment:
Why `port` is a string?

Reply:
It should be int (as in the rest of the connections).

Reply:
Is there a need for `str | int`?

Reply:
No, I don't think that there is a need for both types to be supported. I'll remove the `str` for `port`.